index.html

```diff
@@ -472,6 +472,7 @@
       object contentType;
       SemanticType[] semanticTypes;
       any value;
+      boolean readOnly = false;
```

Comment:
Re-checked ECMAscript convention and will put it to ``writable = false``

https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#Writable_attribute

@sebastiankb we should do the same in the TD